### DISCUSSION TITLE
grafica dei voti cambiata

### DISCUSSION THREAD

**Discussion by manuelbarberi** [iisgiua/giuaschool#425] [Idee e proposte]
Sarebbe bello avere un altro tipo di grafica voti per le famiglie e gli alunni. allego schermata 
![image](https://user-images.githubusercontent.com/96074009/202557545-2a152536-785d-4a33-8565-cf36a7d86004.png)

**Comment by trinko** [comment]
La segno come richiesta.

**Comment by manuelbarberi** [comment]
@trinko per le valutazioni a testo libero come siamo messi?

**Reply by sempliciniluigi89** [reply]
Verissimo,stavo giusto chiedendo la stessa cosa
